python/cudf/cudf/tests/test_reshape.py

```diff
@@ -35,6 +35,9 @@
     pytest_xfail = pytest.mark.skipif
 
 
+rng = np.random.default_rng(seed=0)
+
+
```

Comment:
*nit*: avoid global `rng` and make one in the tests?

Reply:
Fixed it.